asserts/database.go

```diff
@@ -30,22 +30,19 @@ import (
 )
 
 // Backstore is a backstore for assertions. It can store and retrieve
-// assertions by type under primary paths (tuples of strings). Plus it
-// supports more general searches.
+// assertions by type under unique primary key headers. Plus it supports searching by headers.
 type Backstore interface {
-	// Put stores an assertion under the given unique primaryPath.
+	// Put stores an assertion for the given primaryKeyHeaders forming a unique key.
```

Comment:
can't we just use `Put` as the verb in that sentence? `Put stores ...` reads wrong.